### DISCUSSION TITLE
2.0不如1.0功能多，还会更新吗

### DISCUSSION THREAD

**Discussion by leeima** [LIlGG/halo-theme-sakura#433] [General]
### 目前使用的 Sakura 版本？

2.0.0-beta.2

### 目前使用的 halo 版本？

2.8

### 建议/问题

如图，很淡，而且只有一个颜色，无法更改。食用教程不是最新版
![ebabbdaabf0204e134cf05650b916c7](https://github.com/LIlGG/halo-theme-sakura/assets/84554107/365af00d-1bb0-48e2-b18c-0470ae47ff63)
![c35f913d36c776660d20e84c9f1f7f9](https://github.com/LIlGG/halo-theme-sakura/assets/84554107/60662c6b-a9ab-433f-b5e6-b7a307c8f082)

**Comment by LIlGG** [comment]
应当是你 JS 未成功加载，这块是有随机色的。优先取你设置的标签颜色，否则会随机取色，并不是空白。可以查看 https://lixingyong.com/tags

至于主题是会持续更新的，但更新速度可能会相较缓慢

**Comment by leeima** [comment]
希望建个微信交流群
